apps/frontend/src/renderer/components/onboarding/GraphitiStep.tsx

```
@@ -832,25 +832,23 @@ export function GraphitiStep({ onNext, onBack, onSkip }: GraphitiStepProps) {
-                          <li>Better understanding of your codebase over time</li>
-                          <li>Reduces repetitive explanations</li>
-                          <li>No Docker required - uses embedded database</li>
+                          <li>{t('memory.graphitiBenefits.persistent')}</li>
+                          <li>{t('memory.graphitiBenefits.understanding')}</li>
+                          <li>{t('memory.graphitiBenefits.reducesRepetition')}</li>
+                          <li>{t('memory.graphitiBenefits.noDocker')}</li>
                         </ul>
                         <button
                           onClick={handleOpenDocs}
                           className="text-sm text-info hover:text-info/80 flex items-center gap-1"
                         >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)